src/compiler/crystal/tools/doc/html/js/_search.js

```diff
@@ -71,8 +70,8 @@ CrystalDocs.runQuery = function(query) {
       matchedFields.push("name");
     }
 
-    method.args.forEach(function(arg){
-      var argMatches = query.matches(arg.external_name);
+    method.args?.forEach(function(arg){
+      var argMatches = query.matches(arg.external_name ?? arg.name);
```

Comment:
It seems that so far the frontend code never uses modern JavaScript features. So perhaps we should continue to keep them out of here, for better compatibility.

Reply:
[Supported on all modern desktop browsers since the beginning of 2020](https://caniuse.com/mdn-javascript_operators_optional_chaining,mdn-javascript_operators_nullish_coalescing), but I can use more "traditional" methods if compatibility really is a concern

Reply:
I expect it is indeed a concern, but maybe @straight-shoota knows more.

Reply:
I guess, it's probably okay to have that. But on the other hand it's also quite cheap to implement in a legacy-supporting way, and since this would be the only instance of potentially unsupported features, I'd probably prefer going old school for now.